notebooks/noj_book/transformer_references.clj

```diff
@@ -29,7 +32,7 @@
        (kind/md "----------------------------------------------------------")]))))
 
   
-;; ## Transformer reference  - DRAFT 🛠
+;; ## Transformer reference
```

Comment:
More context is needed, like the above.

We cannot expect the reader to know what Transformers are, even if that was mentioned in a paragraph many chapters ago.

This is especially important, since Transformers have different meanings nowadays.